### ISSUE TITLE
using id selectors in Firefox

### ISSUE THREAD

**Issue by gofirst2220** [stylish-userstyles/stylish#333]
see title. I was making a modification of someone else's styles for Facebook and it is giving warning such as "don't use ID selectors" even though I was using them because its required for the style. can this be fixed soon?

**Comment by iigmir** [comment]
I don't think this would be fixed: According to CSSLint, [ID selectors is not recommanded](https://github.com/CSSLint/csslint/wiki/Disallow-IDs-in-selectors).

**Comment by Nekroido** [comment]
One year later. This issue is still not fixed.
